it/src/main/java/org/corfudb/universe/node/server/process/ProcessCorfuServer.java

```diff
@@ -23,12 +24,12 @@
  */
 @Slf4j
 public class ProcessCorfuServer extends AbstractCorfuServer<CorfuServerParams, UniverseParams> {
-    private static final String LOCALHOST = "127.0.0.1";
+    private static final IpAddress LOCALHOST = IpAddress.builder().ip("127.0.0.1").build();
```

Comment:
although localhost might resolve to 127.0.0.1, they are not the same thing, why not just use localhost directly ?

Reply:
Just being consistent. VmCorfuServer uses IP address, also I don't see any reasons to call it ipAddress but provide a DNS name that will be resolved to 127.0.0.1